waiter/src/waiter/client_tools.clj

```diff
@@ -519,13 +519,14 @@
       (str/replace (str test-prefix service-name username (rand-int 3000000)) #"-" ""))))
 
 (defn delete-token-and-assert
-  [waiter-url token]
+  [waiter-url token & {:keys [query-params] :or {query-params {}}}]
```

Comment:
What I mean is, we probably want to automatically excise all tokens in all other tests, instead of just soft-deleting them.